scribble/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
+    redirect_to post_path(@post)
+  end
+
+  def destroy
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.find(params[:id])
+    @comment.destroy
+    redirect_to post_path(@post)
+  end
+
+  private
+    def comment_params
+      params.require(:comment).permit(:commenter, :body)
```

Comment:
Main thing I'd suggest if you were to keep working on this is to set up the relationship between comments and users, just like you did with posts.